### DISCUSSION TITLE
Homebase as doorbell receiver switch

### DISCUSSION THREAD

**Discussion by Brouwertje** [fuatakgun/eufy_security#240] [Ideas]
Hi, in the Eufy app you can turn on or off the chime from the Homebase if you want. (if doorbell pressed) I'd like to use it in automations. (when kid sleeps, no chime) There is an option for ''Indoor Chime'' in the integration, but it's not the setting for the homebase as chime, I think it refers to the setting for the (optional) Eufy doorbell chime (USB dongle) 

I only use the homebase as chime, do you think you can also add a switch for the Homebase chime in the future?

I've added some screens

![Eufy1](https://user-images.githubusercontent.com/89314233/150880055-86c70059-7b6b-4c08-b9bf-f4f42f150345.jpg)
![eufy 2](https://user-images.githubusercontent.com/89314233/150880056-0efbd3c2-0867-4fcb-aef3-3a7742bfdece.jpg)

**Comment by fuatakgun** [comment]
There is an open issue for add on library to fix this. Can you use doorbell chile volume until then?

**Reply by Brouwertje** [reply]
Thanks! I'll look into it.
